src/participant_network.star

```diff
@@ -94,7 +92,9 @@ def launch_participant_network(plan, participants, network_params, global_log_le
 
 	genesis_generation_config_yml_template = read_file(static_files.CL_GENESIS_GENERATION_CONFIG_TEMPLATE_FILEPATH)
 	genesis_generation_mnemonics_yml_template = read_file(static_files.CL_GENESIS_GENERATION_MNEMONICS_TEMPLATE_FILEPATH)
-	total_number_of_validator_keys = network_params.num_validator_keys_per_node * num_participants
+	total_number_of_validator_keys = 0
+	for index, participant in enumerate(participants):
```

Comment:
a pythonic way to do this would be

```py
total_number_of_validator_keys = sum([particiant.valdiator_count for paritcipant in participants])
```

not sure if starlark supports sum though